<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RDishComment</class>
 <widget class="QDialog" name="RDishComment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="leText"/>
      </item>
      <item>
       <widget class="QTableWidget" name="tblVariants"/>
      </item>
      <item>
       <widget class="RKeyboard" name="wKbd" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RKeyboard</class>
   <extends>QWidget</extends>
   <header location="global">rkeyboard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
